language_formatters_pre_commit_hooks/pretty_format_yaml.py

```diff
@@ -46,6 +46,10 @@ def pretty_format_yaml(argv=None):
         with open(yaml_file) as f:
             string_content = ''.join(f.readlines())
 
+        if string_content.strip().startswith("$ANSIBLE_VAULT"):
+            print('Skipping Ansible Vault file {}'.format(yaml_file))
+            continue
```

Comment:
I would suggest to use a different approach in order to make this more generic and not specific to Ansible Vault only.
I would modify line 55 (`yaml.dump(....)`) into something like
```python
content = yaml.load(string_content)
if not isinstance(content, (list, dict)):
    # primitive types will not be altered by the tool
    continue
yaml.dump(content, pretty_content)
```